spec/models/user_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user_1) do
+    User.create(
+      name:     'Joe',
+      email:    'joe@mail.com',
+      password: 'joe_pass',
+      password_confirmation: 'joe_pass'
+    )
+  end
+
+  it 'is valid with a name, email, and password' do
+    expect(user_1).to be_valid
+  end
+
+  it 'is invalid without a name' do
+    user = User.new(name: nil)
+    user.valid?
```

Comment:
If you check that `user.valid?` it should be also included into `expect` for example `expect(user.valid?).to be(false)`

Reply:
fix